session/tidb.go

```diff
@@ -42,6 +43,12 @@ import (
 	"go.uber.org/zap"
 )
 
+type domainInitializer func(
```

Comment:
It seems that it has not been used. Could we remove this code?

Reply:
Yes, I forgot that... :/